spring-cloud-gcp-data-datastore/src/test/java/com/google/cloud/spring/data/datastore/core/DatastoreTemplateTests.java

```diff
@@ -664,6 +714,59 @@ void saveTestCommon(TestEntity parent, boolean lazy) {
     }
   }
 
+  void insertTestCommon(TestEntity parent, boolean lazy) {
```

Comment:
Can you try to reuse the code so that there is less duplication between insert and save?